src/main/scala/com/twitter/diffy/DiffyServiceModule.scala

```diff
@@ -70,6 +70,9 @@ object DiffyServiceModule extends TwitterModule {
   val skipEmailsWhenNoErrors =
     flag[Boolean]("skipEmailsWhenNoErrors", false, "Do not send emails if there are no critical errors")
 
+  var httpsPort =
+    flag[String]("httpsPort", "443", "Port to be used when using HTTPS as a protocol")
```

Comment:
So, what is under https is not the Diffy server, but the services that I am comparing.
I strongly believe proxy.port is a property of the server. And httpsPort is a property for the services.
So can we keep this new flag here?